src/components/Container.js

```diff
@@ -1,28 +1,55 @@
 import React from 'react';
+import Result from './Result';
+import calculate from '../logic/calculate';
 
-const Container = () => (
-  <div className="container">
-    <button type="button" className="col-4" id="result"> 0 </button>
-    <button type="button" className="bg-grey"> AC </button>
-    <button type="button" className="bg-grey"> +/- </button>
-    <button type="button" className="bg-grey"> % </button>
-    <button type="button" className="bg-orange"> + </button>
-    <button type="button" className="bg-grey"> 7 </button>
-    <button type="button" className="bg-grey"> 8 </button>
-    <button type="button" className="bg-grey"> 9 </button>
-    <button type="button" className="bg-orange"> x </button>
-    <button type="button" className="bg-grey"> 4 </button>
-    <button type="button" className="bg-grey"> 5 </button>
-    <button type="button" className="bg-grey"> 6 </button>
-    <button type="button" className="bg-orange"> - </button>
-    <button type="button" className="bg-grey"> 1 </button>
-    <button type="button" className="bg-grey"> 2 </button>
-    <button type="button" className="bg-grey"> 3 </button>
-    <button type="button" className="bg-orange"> + </button>
-    <button type="button" className="col-2 bg-grey"> 0 </button>
-    <button type="button" className="bg-grey"> . </button>
-    <button type="button" className="bg-orange"> = </button>
-  </div>
-);
+export default class Container extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dataObj: { total: null, next: null, operation: null },
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
 
-export default Container;
+  // btnName: null,
+  handleClick = (event) => {
+    this.setState(
+      (prevState) => (
+        {
+          dataObj: calculate(prevState.dataObj, event.target.getAttribute('btn_name')),
+        }
+      ),
+    );
+  }
+
+  // this.setState((prevState) => calculate(prevState, buttonName));
+
+  render() {
+    const { dataObj } = this.state;
+    console.log(dataObj);
+    return (
+      <div className="container">
+        <Result result={dataObj.result} />
```

Comment:
* I think the result is not updated in the state when i click a button redux-logger is updated but not the result kindly fix this 
* Also i get this error in the console kindly make the necessary adjustments 
![image](https://user-images.githubusercontent.com/52497246/130817305-8d9a097e-8d54-4a5d-8f55-3af32504dbb9.png)